cli/command/plugin/inspect.go

```diff
@@ -37,20 +34,8 @@ func newInspectCommand(dockerCli *command.DockerCli) *cobra.Command {
 func runInspect(dockerCli *command.DockerCli, opts inspectOptions) error {
```

Comment:
https://github.com/docker/docker/blob/master/docs/reference/commandline/plugin_inspect.md needs to be updated.